easy-coding-standard.yaml

```diff
@@ -1,5 +1,11 @@
 imports:
     - { resource: '%vendor_dir%/lmc/coding-standard/easy-coding-standard.yaml' }
 services:
+    PhpCsFixer\Fixer\ClassNotation\VisibilityRequiredFixer:
```

Comment:
Nice suggestion! I've added this rule to the parent codestyle - could you now please remove this from the PR? (As it  will became redundant once I tag `lmc/coding-standard`)

Reply:
Done!